runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -516,16 +515,16 @@ private void onWidgetLoaded(WidgetLoaded event)
 				container = client.getItemContainer(InventoryID.CHAMBERS_OF_XERIC_CHEST);
 				chestLooted = true;
 				break;
-			case (WidgetID.THEATRE_OF_BLOOD_GROUP_ID):
+			case (WidgetID.THEATRE_OF_BLOOD_REWARD_GROUP_ID):
 				if (chestLooted)
 				{
 					return;
 				}
-				int region = WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID();
-				if (region != THEATRE_OF_BLOOD_REGION)
+				if (!(client.getLocalPlayer().getWorldLocation().getRegionID() == THEATRE_OF_BLOOD_REGION))
```

Comment:
`if (getRegionID() != THEATRE_OF_BLOOD_REGION)`

Reply:
@ThatGamerBlue is that what you meant? still havent gotten intellij to stop using gradle 5 so edited directly in github